src/spikeinterface/widgets/widget_list.py

```diff
@@ -10,10 +10,9 @@
 from .autocorrelograms import AutoCorrelogramsWidget
 from .crosscorrelograms import CrossCorrelogramsWidget
 from .isi_distribution import ISIDistributionWidget
-from .motion import DriftRasterMapWidget, MotionWidget, MotionInfoWidget
+from .motion import MotionWidget
```

Comment:
I think you based your PR on a previous version of `main` where these widgets wee not implemented yet. Can you revert?

Reply:
Sure, I will fix those lines!